packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -64,6 +64,11 @@ export const buttonView = ( {
 		border: 0;
 	}
 
+	&[disabled] {
+		opacity: 0.4;
+		cursor: default;
```

Comment:
We have more and more of these disabled styles that do `opacity: 0.4`. Perhaps this is a good opportunity to generalize at some point.

Reply:
Agreed, although this PR is not the right place for it. We should probably assess the current state of shared config in the package, and decide next steps. For example, some variables come from the `base-styles` package, some are internal to the components package. Some variables are defined in Sass, some as CSS custom properties, and others in a JS object.

Reply:
Definitely should be a separate PR 👍